src/selectors/getCurrentCompiledProjectKey.js

```diff
@@ -0,0 +1,9 @@
+import get from 'lodash/get';
+
+export default function getCurrentCompiledProjectKey(state) {
+  return get(
+    state.get('compiledProjects').last(),
+    'timestamp',
+    0,
```

Comment:
Do we want a default of 0 here? I think `undefined` is probably more correct

Reply:
No we do not :)